public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
Are we disallowing anything here?

Reply:
Not yet, its a place holder file added by the snowpack bootstrap

Reply:
May we comment it out then?